### PR TITLE
Rename Combining service to AccommodationMergingService

### PR DESCRIPTION
**Pull request by oleg1std**
Rename Combining service to AccommodationMergingService

### CODE REVIEW

File: HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs [hunk 1]
```diff
@@ -26,15 +27,15 @@ public class CombiningService : ICombiningService
...
+        public async Task<Result> MergeUncertain(int uncertainMatchId, CancellationToken cancellationToken = default)
         {
             var uncertainMatch =
                 await _context.AccommodationUncertainMatches.SingleOrDefaultAsync(um => um.Id == uncertainMatchId && um.IsActive, cancellationToken);
 
             if (uncertainMatch == default)
                 return Result.Success();
 
-            var (_, isFailure, error) = await Combine(uncertainMatch.SourceHtId, uncertainMatch.HtIdToMatch);
+            var (_, isFailure, error) = await Merge(uncertainMatch.SourceHtId, uncertainMatch.HtIdToMatch);
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs:38]: Db columns also needed to change

File: HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs [hunk 2]
```diff
@@ -57,32 +58,34 @@ public async Task<Result> CombineUncertain(int uncertainMatchId, CancellationTok
...
-        /// <param name="baseHtAccommodationId"></param>
-        /// <param name="combinedHtAccommodationId"></param>
+        /// <param name="request"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
-        public async Task<Result> Combine(string baseHtAccommodationId, string combinedHtAccommodationId, CancellationToken cancellationToken = default)
+        public async Task<Result> Merge(MergeAccommodationsRequest request, CancellationToken cancellationToken = default)
         {
+            var baseHtAccommodationId = request.BaseHtAccommodationId;
+            var mergedHtAccommodationId = request.MergedHtAccommodationId;
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs:67]: htAccommodationToMerge

File: HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs [hunk 3]
```diff
@@ -93,15 +96,15 @@ public async Task<Result> Combine(string baseHtAccommodationId, string combinedH
...
+            await _accommodationChangePublisher.PublishRemoved(mergedAccommodationId);
 
             await _accommodationChangeLogCollectorService.SaveCollectedEventLogs(cancellationToken);
             return Result.Success();
         }
 
 
         // This method only make changes on db context - not on db.
-        private async Task<Result> Combine(int sourceHtId, int htIdToMatch)
+        private async Task<Result> Merge(int sourceHtId, int htIdToMatch)
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs:107]: htIdToMerge

File: HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs (hunk 2 above)
Comment (oleg1std) [reply HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs:67]: I think 'merged' at the beginning of the variable looks more descriptive

File: HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs (hunk 1 above)
Comment (oleg1std) [reply HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs:38]: could you please be more specific

Comment (hasmikharutyunyan) [reply HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs:38]: HtIdToMatch change to `HtIdToMerge`

File: HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs (hunk 2 above)
Comment (hasmikharutyunyan) [reply HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs:67]: `merged` is described that accommodations already  merged, I think it is not correct.

File: HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs (hunk 3 above)
Comment (oleg1std) [reply HappyTravel.Nakijin.Api/Services/ManagementServices/Accommodations/AccommodationMergingService.cs]: fixed

**Review by hasmikharutyunyan — APPROVED** [review]

**Pull request merged**